rspace/src/main/scala/coop/rchain/rspace/trace/Event.scala

```diff
@@ -43,7 +43,7 @@ object COMM {
 
 sealed trait IOEvent extends Event
 
-class Produce private (val hash: Blake2b256Hash) extends IOEvent {
+class Produce private (val channelsHash: Blake2b256Hash, val hash: Blake2b256Hash) extends IOEvent {
```

Comment:
curious: couldn't this be a `case class`?

Reply:
I don't know why, but historically Produce and Consume were not case classes, while COMM was. Definitely all of them are good candidates to be case classes. Thanks for pointing this out.